File: DAL/Models/Devises.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DAL.Models
{
   public class Devises:AuditableEntity
    {
        public Devises()
        {
            this.Fournisseurs =new HashSet<Fournisseurs>();
        }
        public int Id { get; set; }
        public string Symbole { get; set; }
        public string Label { get; set; }
        public string CodeIso { get; set; }

        public ICollection<Fournisseurs> Fournisseurs { get; set; }
    }

}

File: Atomics Manager/ClientApp/src/app/admin/models/devises.ts
export class Devises {

    constructor(symbole?:string,label?:string,codeIso?:string)
    {
        this.symbole=symbole;
        this.label=label;
        this.codeIso=codeIso;
    }
    public id:number;
    public symbole:string;
    public label:string;
    public codeIso:string;
}
